src/index.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import App from './components/App';
+
+import './index.scss';
```

Comment:
i think split `css` files can be great but i would like to have the root one be called `global.scss`

mainly because it makes more sense that way..

and then we should have each component have their own `.scss` file